### PR TITLE
Support testing `content` properties which include a curly brace.

### PR DESCRIPTION
**Pull request by notlee**
True parses braces within the content property as a block. E.g.
```css
	html:before {
        content: '{ "a": 1, "b": 2 }';
    }
```

To prevent this, only match a closing brace as the end of a block
provided it is not followed by a comma, quote, or double quote.

An example where this kind of CSS syntax might be used is to share
values between JS and CSS (where Custom Properties can't be used).
https://css-tricks.com/making-sass-talk-to-javascript-with-json/#article-header-id-0

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,13067 @@
+{
```
Comment (jgerigmeyer) [inline package-lock.json]: Please remove this file from the PR.

Comment (notlee) [reply package-lock.json]: Done 👍

**Comment by notlee** [comment]
Thanks for the speedy review 👏

**Comment by mirisuzanne** [comment]
Thanks!

**Pull request merged**